### DISCUSSION TITLE
Push to NPM

### DISCUSSION THREAD

**Discussion by ghostandthemachine** [mkkellogg/GaussianSplats3D#95] [General]
Hi @mkkellogg, this is awesome! I'm trying to get things running via NPM but it appears the latest on NPM (from 11 days ago at the moment) is out of sync with your demos. When trying to run the example code the viewer.addSplatScene is not present. The viewer is there and has other functions but not that one so my suspicion is that it's just a version issue.

Cheers

**Comment by mkkellogg** [comment]
Thanks for the heads up! I went ahead and updated the NPM package to match the repo. Let me know if that solves your issue.
